docs/csharp/how-to/modify-string-contents.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "How to: Modify String Contents (C# Guide)"
+ms.date: 02/26/2018
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+helpviewer_keywords: 
+  - "strings [C#], modifying"
+author: "BillWagner"
+ms.author: "wiwagn"
+---
+
+# How to: Modify String Contents (C# Programming Guide)
+
+This article demonstrates several techniques to produce a `string` by modifying an existing `string`. All the techniques demonstrated return the result of the modifications as a `string` object. To clearly demonstrate this, the examples all store the result in a new variable. You can then examine both the original `string` and the `string` resulting from the modification when you run each example. 
```

Comment:
This is a much better opening paragraph, Bill! I'd modify the second sentence just a bit to stress that a new string is returned -- "as a new \`string\` object."